osu.Game/Storyboards/Drawables/DrawableStoryboardSprite.cs

```diff
@@ -114,13 +112,13 @@ public DrawableStoryboardSprite(StoryboardSprite sprite)
         }
 
         [BackgroundDependencyLoader]
-        private void load(IBindable<WorkingBeatmap> beatmap, TextureStore textureStore)
+        private void load(TextureStore textureStore, Storyboard storyboard)
         {
-            var path = beatmap.Value.BeatmapSetInfo?.Files?.Find(f => f.Filename.Equals(Sprite.Path, StringComparison.OrdinalIgnoreCase))?.FileInfo.StoragePath;
-            if (path == null)
-                return;
+            var drawable = storyboard.CreateSpriteFromResourcePath(Sprite.Path, textureStore);
+
+            if (drawable != null)
+                InternalChild = drawable.With(s => s.Anchor = s.Origin = Anchor.Centre);
```

Comment:
this is completely wrong, and ignored any origin changes applied to the sprite

Reply:
ouch. not sure where i got that from...